pom.xml

```diff
@@ -59,6 +61,11 @@
                 <scope>import</scope>
                 <type>pom</type>
             </dependency>
+            <dependency>
+                <groupId>io.gravitee.common</groupId>
+                <artifactId>gravitee-common</artifactId>
+                <version>${gravitee-common.version}</version>
+            </dependency>
```

Comment:
provided ?

Reply:
It is already in scope `provided` (here it is the `dependencyManagement` 🙂)